tools/fp20compiler/ts1.0_inst_list.cpp

```diff
@@ -46,8 +46,10 @@ void InstList::Invoke()
     printf("pb_push1(p, NV097_SET_SHADER_OTHER_STAGE_INPUT,\n    ");
     for (i=1; i<size; i++) {
         if (i != 1) printf("    | ");
-        int previousTexture = (int)list[i].args[0];
-        assert(i > previousTexture);
+        int previousTexture = 0;
+        if (list[i].opcode.bits.dependent)
+            previousTexture = (int)list[i].args[0];
+        //assert(i > previousTexture);

```

Comment:
This assert must be removed for error reporting (otherwise this triggers first, before errors are printed)